microsoft-edge/origin-trials/index.md

```diff
@@ -1,26 +1,172 @@
+* A single domain, without supporting its subdomains.  For example, register `https://example.com`, and in the **Subdomain Support** section, select the **No** option button.
+* A single subdomain.  For example, register `https://beta.example.com`, and in the **Subdomain Support** section, select the **No** option button.
+* A domain and its subdomains.  For example, register `https://example.com`, and in the **Subdomain Support** section, select the **Yes** option button.
+<!-- todo: check section heading & option button labels at microsoft.com portal; the above ui is from github.io portal -->
+
+
+<!-- ---------- -->
+###### Paths in URLs, and query parameters
+
+In URLs, paths and query parameters aren't supported by the origin trial system.  When registering for an origin trial, if a URI is submitted that has a trailing path or a query parameter, such as `https://example.com/path/new-feature`, the root domain (`https://example.com`) will be registered, without the trailing path or query parameter.
+
+
+<!-- ------------------------------ -->
+#### Duration of an origin trial
+
+
+<!-- ---------- -->
+###### A scheduled expiration date
+
+Once you have registered for a trial, your token will be valid until the expiration date that's listed on the origin trial's informational card.  After this date, the token will no longer function, and the feature that's associated with the trial will be considered ready to ship, or will be brought back<!-- todo: clarify "brought back" - removed from OT portal? --> for further development based on developer feedback. 
+
+
+<!-- ---------- -->
+###### Early end
+
+A browser feature team may decide to early-end an experiment; for example:
+* If a security incident is caused by the feature.
+* If the feature team has collected enough feedback and has concluded that the currently proposed API doesn't meet the web developer needs, and a major redesign needs to happen.
+
+If an experiment ends early, the feature team that owns the origin trial sends emails to all developers who are enrolled in the experiment. 
+
+
+<!-- ------------------------------ -->
+#### Opting out of an experiment
+
+To opt out of an experiment that you're enrolled in, remove the origin trial token from the `<meta>` tag or from the server response headers, shown in [How to use an origin trial token](#how-to-use-an-origin-trial-token), above.
+
+
+<!-- ------------------------------ -->
+#### Best practices for experimental feature implementation
+
+Treat origin trials as experimental.  Provide graceful fallback mechanisms in case the origin trial token expires, or in case the browser feature team ends the origin trial.
+
+For best practices for feature detection, see [Implementing feature detection](https://developer.mozilla.org/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) at MDN.
 
 
 <!-- ====================================================================== -->
 ## See also
```

Comment:
added 
* [Get started with origin trials](https://developer.chrome.com/docs/web-platform/origin-trials) - Chrome docs.
ok?  @leahmsft

Reply:
keeping link, for review